bench/wtperf/idle_table_cycle.c

```diff
@@ -41,9 +41,10 @@ check_timing(WTPERF *wtperf, const char *name, struct timespec start, struct tim
     last_interval = (uint64_t)(WT_TIMEDIFF_SEC(*stop, start));
 
     if (last_interval > opts->idle_table_cycle) {
-        lprintf(wtperf, ETIMEDOUT, 0, "Cycling idle table failed because %s took %" PRIu64
-                                      " seconds which is longer than configured acceptable"
-                                      " maximum of %" PRIu32 ".",
+        lprintf(wtperf, ETIMEDOUT, 0,
```

Comment:
@keithbostic String literal handling has changed a bit. I believe this addresses WT-5936. Do you agree?

Reply:
No, I don't agree. The problem in WT-5936 is that strings are oddly broken, and this isn't changing that, it's just moving the oddly broken strings to different places without joining the strings where it makes sense.

Reply:
Ah, I understand now. It's because the strings aren't filled out to the full column length (similar to the comments situation).